### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by BenMukebo**
In this feature, we have displayed something on the page

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const mainDiv = document.querySelector('.main-div');
+
+
```
Comment (hiromataba) [inline index.js]: I think you should not leave a lot spaces

File: index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+ const output = mainDiv.innerHTML = `<p>Fuck you</p>`
```
Comment (hiromataba) [inline index.js]: Avoid hard words

**Review by hiromataba — CHANGES_REQUESTED** [review]
### Dear Ben,

I think you should check my comments below and make the required changes

Thank you

Best,
Hiroi

File: index.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+const mainDiv = document.querySelector('.main-div');
+const output = mainDiv.innerHTML = `<p>Hello World</p>`
+mainDiv.appendChild(output)
```
Comment (hiromataba) [inline index.js:3]: Thanks for making the changes

**Review by hiromataba — APPROVED** [review]
Congratulations Ben

Your project is now ready to be merged 

thanks

**Comment by BenMukebo** [comment]
https://us05web.zoom.us/j/87853239993?pwd=QzBGTkV1TkdoODBxOVBFd3dhNVZlUT09

**Pull request merged**